test/test-network/systemd-networkd-tests.py

```diff
@@ -2724,14 +2725,27 @@ def test_dhcp_client_ipv6_only(self):
 
         output = check_output(*networkctl_cmd, '-n', '0', 'status', 'veth99', env=env)
         print(output)
-        self.assertRegex(output, '2600::')
+        self.assertRegex(output, '2600::1a:2b:3c:4d')
         self.assertNotRegex(output, '192.168.5')
 
         # Confirm that ipv6 token is not set in the kernel
         output = check_output('ip token show dev veth99')
         print(output)
         self.assertRegex(output, 'token :: dev veth99')
 
+    def test_dhcp_client_ipv6_token_prefixstable(self):
+        copy_unit_to_networkd_unit_path('25-veth.netdev', 'dhcp-server-veth-peer.network', 'dhcp-client-ipv6-token-prefixstable.network')
```

Comment:
I think we should name this function to token only and the file config file also. And add another file as this and do test for prefix stable.

```
➜  network cat veth0.network 
[Match]
Name=veth0

[Network]
IPv6PrefixDelegation=yes
Address=2001:888:0db8:1::a/64


[IPv6Prefix]
Prefix=2001:888:0db8:1::/64

➜  network cat veth99.network
[Match]
Name=veth99

[Network]
DHCP=no
IPv6Token=prefixstable:2001:888:0db8:1::/64

[Match]
Name=veth99

[Network]
DHCP=no
IPv6AcceptRA=yes
IPv6Token=prefixstable:2001:888:0db8:1::/64
```

Reply:
Thank you for the advice. Test is updated now.